app/js/components/service_form.test.js

```diff
@@ -0,0 +1,74 @@
+jest
+    .dontMock('fs')
+    .dontMock('jquery');
+
+
+window.$ = require('jquery');
+
+
+const loadServiceForm = require('./service_form.ts');
+
+describe('validateToggles serviceEditForm', function() {
```

Comment:
This description doesn't sit well with me. Or is this Jest convention?

Suggestion:
`validate visibility toggling of service status fields on the service edit form`

Reply:
Thanks for the good suggestion, I've changed it accordingly.